src/features/game/escape/GameEscapeManager.ts

```diff
@@ -200,15 +206,12 @@ class GameEscapeManager implements IGameUI {
    * Escape Menu is responsible in contacting various managers to apply the settings.
    */
   private async applySettings() {
-    const sfxVol = this.sfxVolumeRadioButtons
-      ? parseFloat(this.sfxVolumeRadioButtons.getChosenChoice())
-      : 1;
-    const bgmVol = this.bgmVolumeRadioButtons
-      ? parseFloat(this.bgmVolumeRadioButtons.getChosenChoice())
-      : 1;
+    const sfxVol = this.sfxVolumeSlider ? this.sfxVolumeSlider.getValue() : 1;
+    const bgmVol = this.bgmVomlumeSlider ? this.bgmVomlumeSlider.getValue() : 1;
+    const muted = this.muteButton ? this.muteButton.getChoice() : false;
 
     // Save settings
-    const newSettings = { bgmVolume: bgmVol, sfxVolume: sfxVol };
+    const newSettings = { bgmVolume: bgmVol, sfxVolume: sfxVol, isMuted: muted };
```

Comment:
Would be nice muting behavior was as follows:
- Mute button placed below both sliders so it doesn't appear visually tied to just SFX or BGM
- isMuted is simply the state of bgmVolume and sfxVolume being 0 so it wouldn't need a separate field. In the UI, selecting the checkbox will simply set the sliders to 0. Similarly setting both sliders to 0 manually will activate the checkbox

What do you think?